admin_guide/allocating_node_resources.adoc

```diff
@@ -155,6 +164,103 @@ $ curl <certificate details> https://<master>/api/v1/nodes/cluster.node22/proxy/
+
+.Node Cgroup Settings
+====
+[source,yaml]
+----
+kubeletArguments:
+  cgroups-per-qos:
+    - "true" <1>
+  cgroup-driver:
+    - "systemd" <2>
+  enforce-node-allocatable:
+    - "pods" <3>
+----
+<1> Enable or disable the new cgroup hierarchy managed by the node.  Any change
+of this setting requires a full drain of the node.  This flag must be true to allow the node to
+enforce node allocatable.  We do not recommend users change this value.
+<2> The cgroup driver used by the node when managing cgroup hierarchies.  This
+value must match the driver associated with the container runtime.  Valid values
+are `systemd` and `cgroupfs`.  The default is `systemd`.
+<3> A comma-delimited list of scopes for where the node should enforce node
+resource constraints.  Valid values are `pods`, `system-reserved`, and `kube-reserved`.
+The default is `pods`.  We do not recommend users change this value.
+====
+
+Optionally, the node can be made to enforce kube-reserved and system-reserved by
+specifying those tokens in the enforce-node-allocatable flag.  If specified, the
+corresponding `--kube-reserved-cgroup` or `--system-reserved-cgroup` needs to be provided.
+In future releases, the node and container runtime will be packaged in a common cgroup
+separate from `system.slice`.  Until that time, we do not recommend users
+change the default value of enforce-node-allocatable flag.
+
+Administrators should treat system daemons similar to Guaranteed pods.  System daemons
+can burst within their bounding control groups and this behavior needs to be managed
+as part of cluster deployments.  Enforcing system-reserved reservations
+can lead to critical system services being CPU starved or OOM killed on the node. The
+recommendation is to enforce system-reserved only if operators have profiled their nodes
+exhaustively to determine precise estimates and are confident in their ability to 
+recover if any process in that group is OOM killed.
+
+As a result, we strongly recommended that users only enforce node allocatable for
+`pods` by default, and set aside appropriate reservations for system daemons to maintain
+overall node reliability.
+
+[[node-enforcement]]
+== Eviction Thresholds
+
+If a node is under memory pressure, it can impact the entire node and all pods running on
+it.  If a system daemon is using more than its reserved amount of memory, an OOM
+event may occur that can impact the entire node and all pods running on it.  To avoid
+(or reduce the probability of) system OOMs the node
+provides xref:../admin_guide/out_of_resource_handling.adoc[Out Of Resource Handling].
+
+By reserving some memory via the `--eviction-hard` flag, the node attempts to evict
+pods whenever memory availability on the node drops below the reserved value.
+If system daemons did not exist on a node, pods are limited to the memory
+`capacity - eviction-hard`. For this reason, resources reserved for evictions are not
```

Comment:
s/reserved for evictions/set aside as a buffer for eviction before reaching out of memory conditions/